doc/api/permissions.md

```diff
@@ -540,6 +540,24 @@ Wildcards are supported too:
 * `--allow-fs-read=/home/test*` will allow read access to everything
   that matches the wildcard. e.g: `/home/test/file1` or `/home/test2`
 
+##### Accessing files with comma in path
+
+To access files with comma in path you can change the path delimiter using the
```

Comment:
```suggestion
To access files with a comma in the path you can change the path delimiter using the
```

Reply:
This will need line changes not to exceed 80 characters, I will change it